### PR TITLE
job posting page

### LINKED ISSUE #16: Job Posting Creation UI for Employer
**Description**
create a webpage to allow Employer to create a job posting and input the details for it. The detailed info includes: Title, Location, Exp. Level(Coop, Intern, Junior, Mid, Senior), Description, Required Skill Set(Java, Python, …), Posting time(auto), Salary.
story points:3
**Related User Story**
#5

### PR DESCRIPTION
**Pull request by AutoSleep**

### CODE REVIEW

**Comment by AutoSleep** [comment]
The employer can post job now

**Comment by AutoSleep** [comment]
image
image

**Review by meyruiz — DISMISSED** [review]

File: fe-career-platform/src/pages/JobPosting/JobPosting.jsx [hunk 1]
```diff
@@ -71,7 +68,7 @@ export default function JobPosting() {
             <div className='header'>
                 <h1>List of Job Posting</h1>
                 {!openJobPostingSection && (
-                    <Button variant="contained" onClick={handleJobPostingSection}>
+                    <Button className="button-right" variant="contained" onClick={handleJobPostingSection}>
```
Comment (meyruiz) [inline fe-career-platform/src/pages/JobPosting/JobPosting.jsx:85]: Can you add a check to enable this functionality only for employers?

**Review by Alor-e — APPROVED** [review]
Looks good to me.

**Review by meyruiz — COMMENTED** [review]
Looks good! Ideally we would redirect the user to another page if it's not an employer, but since this is a prototype it is fine.

**Pull request merged**